.github/workflows/ci.yml

```diff
@@ -93,7 +93,7 @@ jobs:
 
       - name: Clone CCI
         run: |
-          git clone https://qchateau:$GITHUB_TOKEN@github.com/conan-center-bot-testing/conan-center-index.git
+          git clone https://${{ github.repository_owner }}:$GITHUB_TOKEN@github.com/conan-center-bot-testing/conan-center-index.git
```

Comment:
the only "solution" to this would be to define the test repo in an environment variable at the github settings level

Reply:
Yeah, I guess that would be fine. To make it easier we could use a default value for this variable.
On a similar subject, we probably want env vars for the name and email of the git user